mia/helpers/utils.py

```diff
@@ -327,40 +327,51 @@ def urlretrieve(url, install_filepath, cache_path=None):
     (Caching not yet implemented.)
     """
 
-    cache_enabled = False if (cache_path == None) else True
+    # Create the arguments list for wget.
+    wget_arguments = ['--no-verbose', '--server-response']
 
-    if cache_enabled:
-        filename = url.split('/')[-1]
-        download_filepath = os.path.join(cache_path, filename)
-    else:
+    if cache_path is None:
+        cache_enabled = False
         download_filepath = install_filepath
+    else:
+        cache_enabled = True
+        wget_arguments.append('--continue')
```

Comment:
If this is our way around the index.xml issue, I'm a-ok with this being a temporary resolution, but just lemme know so I can open another issue for doing it a little more intentionally and reverting this later :)